precipitation/ngsild-payloads/APIX.json

```diff
@@ -0,0 +1,15 @@
+{
+    "id": "urn:ngsi-ld:dataSource:APIMeteoConcept",
+    "type": "dataSource",
```

Comment:
'type' value starts with uppercase and I think that it have to be defined in the context files